### DISCUSSION TITLE
Debugging the node graph?

### DISCUSSION THREAD

**Discussion by Hyperlooper** [FlowiseAI/Flowise#352] [Q&A] [answered]
Hello, first of all, this is a great tool and I really appreciate everyone's efforts on it. I was wondering if there is any way of seeing the output values of individual nodes in your graph for debugging purposes? I am finding it difficult to reason about the output of graphs as a whole, because I can't 'follow' what values the individual nodes are outputting, and this makes debugging a bit tough.

**Comment by HenryHengZJ** [comment, marked as answer]
will be adding logger soon to output each node result to logs - https://github.com/FlowiseAI/Flowise/pull/358

**Reply by Hyperlooper** [reply]
That will be a fantastic step!

**Reply by warlocc** [reply]
Did this ever get added?
